app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1> Ingredient List: </h1>
+<% @ingredients.each do |ingredient| %>
+  <h3> <%= ingredient.name %> </h3>
+  <p> <%= ingredient.cost %> </p>
+<% end %> 
```

Comment:
Just note that there was a specific format for how the ingredients names and cost were supposed to be listed. `Name: Cost`.